src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -36,6 +36,7 @@ public HelpWindow(Stage root) {
         super(FXML, root);
         helpMessage.setText(HELP_MESSAGE);
         root.sizeToScene();
```

Comment:
btw, please check if this line is needed anymore. I'm guessing it was added to override the "maximize" behavior, but it may not actually be needed once we set `maximize="false"` since according to the docs at https://docs.oracle.com/javase/8/javafx/api/javafx/scene/Scene.html JavaFX is supposed to, by default, size scenes to the size of their content.